lxd/instance/drivers/driver_lxc.go

```diff
@@ -6017,6 +6017,11 @@ func (d *lxc) insertMountLXD(source, target, fstype string, flags int, mntnsPID
 		target = "/" + target
 	}
 
+	idmappedStr := fmt.Sprintf("%s", idmap.IdmapStorageNone)
+	if shiftfs {
```

Comment:
I think this commit `43bcab46d67cb3c14c49cb0be18f83a1f2b9659b` needs a rebase as it looks like this part has been removed later on when the function definition removes the `shiftfs` bool argument.

Reply:
I think that just needs to be removed.